jwks_pairs.go

```diff
@@ -0,0 +1,36 @@
+package jwtauth
+
+import (
+	"github.com/mitchellh/mapstructure"
+)
+
+type JWKSPair struct {
+	JWKSUrl   string `mapstructure:"jwks_url"`
+	JWKSCAPEM string `mapstructure:"jwks_ca_pem"`
+}
+
+func NewJWKSPairsConfig(jc *jwtConfig) ([]*JWKSPair, error) {
+	if len(jc.JWKSPairs) <= 0 {
```

Comment:
I often use `<=` instead of `==` as it is more restrictive than `==`. Though `len` should always return a positive integer so I'll defer to your preference.

Reply:
This is probably an overly defensive habit so as not to assume that the function called always returns a positive number.

Reply:
Makes sense! I think it's okay to leave as is :)